tests/__init__.py

```diff
@@ -0,0 +1 @@
+from . import testing_utils
```

Comment:
Does adding this `__init__.py` to the tests dir result in it getting picked up and installed as a package (called `tests`) based on `find_packages` used in the setup.py script? You can check by pip installing the package (no -e) and then seeing if a `tests` dir has shown up in your python site-packages dir (where this is depends on python install method.

Reply:
Good catch, that's exactly what happens in this case -- will remove the `__init__.py`.